onnxruntime/python/tools/kernel_explorer/kernels/fast_gelu_test.py

```diff
@@ -0,0 +1,100 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import sys
+
+sys.path.append("../build")
+
+import kernel_explorer as ke
+import numpy as np
+import pytest
+
+
+def dtype_to_funcs(dtype):
+    type_map = {
+        "float16": list(filter(lambda x: "FastGelu_half" in x, dir(ke))),
+        "float32": list(filter(lambda x: "FastGelu_float" in x, dir(ke))),
+        "float64": list(filter(lambda x: "FastGelu_double" in x, dir(ke))),
+    }
+    return type_map[dtype]
+
+
+def fast_gelu(x, bias):
```

Comment:
Considering there will be kernels be much more complex than this one, we better to reuse existing kernel in ORT as the reference impl?

Reply:
Yes, we may use a CPU ORT implementation as a reference for some kernels. For kernels like Gemm or layer norm, a reference implementation in numpy is only a few lines of code, so it might be ok.

Reply:
This will incur some additional verification effort if we implement them again. But anyway, lets choose to move fast first.